renovate.json5

```diff
@@ -46,6 +46,37 @@
         "react-docgen",
       ],
     },
+    {
+      // minor updates in packages <1.0.0 - need master issue approval
+      // not grouped
+      masterIssueApproval: true,
+      updateTypes: ["minor"],
+      packageNames: [
+        // below is list of packages that use 0.X version range, any minor bump there can contain breaking changes, so we just ignore minor bumps for those packages and will need to bump them manually
+        "@reach/skip-nav",
+        "@theme-ui/prism",
+        "@theme-ui/typography",
+        "axios",
+        "babel-preset-gatsby",
```

Comment:
Would probably remove this since it's in the monorepo but I see that it could be useful for stuff in the repo beside packages like e2e tests so I'll leave this to you 
```suggestion

```